### PR TITLE
Create Integration Entity With Duplicated Name Test

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
**Description**
This PR with a test that creates Integration with Duplicated Name

**Steps:**

1. Log in to backend as admin user
2. Navigate to System > Extensions > Integrations
3. Click 'Add New Integration'
4. Fill in all required data
5. Click "Save" button to save Integration1
6. Click 'Add New Integration'
7. Fill in all required data and use the same name as for Integration1
8. Click "Save" button
9. Perform all assertions
**Fixed Issues:**
#517 CreateIntegrationWithDuplicatedNameTest to MFTF

### CODE REVIEW

File: app/code/Magento/Integration/Test/Mftf/Test/AdminCreateIntegrationEntityWithDuplicatedNameTest.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            <group value="mtf_migrated"/>
+        </annotations>
+        <before>
+            <actionGroup ref="LoginAsAdmin" stepKey="LoginAsAdmin"/>
+        </before>
+        <after>
+            <actionGroup ref="logout" stepKey="logout"/>
+        </after>
+
+        <!-- TEST BODY -->
```
Comment (lbajsarowicz) [inline app/code/Magento/Integration/Test/Mftf/Test/AdminCreateIntegrationEntityWithDuplicatedNameTest.xml]: I think you don't need comments there :-)

File: app/code/Magento/Integration/Test/Mftf/Test/AdminCreateIntegrationEntityWithDuplicatedNameTest.xml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        <!-- Navigate To Integrations Page -->
+        <actionGroup ref="AdminNavigateMenuActionGroup" stepKey="navigateToIntegrationsPage">
+            <argument name="menuUiId" value="{{AdminMenuSystem.dataUiId}}"/>
+            <argument name="submenuUiId" value="{{AdminMenuSystemExtensionsIntegrations.dataUiId}}"/>
+        </actionGroup>
+        <!-- Click the "Add New Integration" button -->
```
Comment (lbajsarowicz) [inline app/code/Magento/Integration/Test/Mftf/Test/AdminCreateIntegrationEntityWithDuplicatedNameTest.xml]: This also is deducted from stepKey :-)

File: app/code/Magento/Integration/Test/Mftf/Test/AdminCreateIntegrationEntityWithDuplicatedNameTest.xml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        <actionGroup ref="AdminNavigateToCreateIntegrationPageActionGroup" stepKey="clickAddNewIntegrationButton"/>
+        <!-- Create New Integration -->
```
Comment (lbajsarowicz) [inline app/code/Magento/Integration/Test/Mftf/Test/AdminCreateIntegrationEntityWithDuplicatedNameTest.xml]: This also is deducted from stepKey :-)

**Comment by DmitryTsymbal** [comment]
Hello @lbajsarowicz ! Thank you for the review.
I have deleted comments.
Please check it.
Thanks

**Pull request merged**